build/deps/markdownlint.Dockerfile

```diff
@@ -1 +1 @@
-FROM ghcr.io/igorshubovych/markdownlint-cli:v0.32.2
+FROM davidanson/markdownlint-cli2:latest
```

Comment:
```suggestion
FROM davidanson/markdownlint-cli2:v0.5.1
```

We use dependabot to track updates explicitly

Reply:
Sorry about that 🥲👍🏽